ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/logging_settings.yml

```
@@ -0,0 +1,86 @@
+---
+type: spine
+spine:
+  nodes:
+    - name: logging_settings
+      mgmt_ip: 192.168.66.22/24
+      id: 1
+      loopback_ipv4_pool: 1.2.3.4/24
+      bgp_as: 65000
+      inband_mgmt_vrf: inband_mgmt_vrf
+      inband_mgmt_interface: Vlan123
+      mgmt_interface: Vlan121
+mgmt_interface_vrf: MGMT
+logging_settings:
+  console: errors
+  monitor: disabled
+  buffered:
+    size: 1000000
+    level: warnings
+  repeat_messages: true
+  trap: disabled
+  synchronous:
+    level: warnings
+  format:
+    timestamp: traditional year timezone
+    rfc5424: true
+    hostname: fqdn
+  hosts:
+    - name: 10.10.10.7
+      vrf: mgt0
+    - name: 2001:db8::10:7
+    - name: 30.30.30.7
+      protocol: tcp
+      ports:
+        - 100
+        - 200
+    - name: 2001:db8::30:7
+      protocol: tcp
+      ports:
+        - 100
+        - 200
+    - name: 40.40.40.7
+      vrf: vrf_with_no_source_interface
+      ports:
+        - 300
+        - 400
+    - name: 2001:db8::40:7
+      vrf: default
+      ports:
+        - 300
+        - 400
+    - name: sslhost.net
+      vrf: use_mgmt_interface_vrf
+      protocol: tls
+      ssl_profile: logging-ssl
+      ports:
+        - 6515
+    - name: syslog.server.com
```

please don't use official domains here. (same elsewhere)
```suggestion
    - name: syslog.server.local
```

---

updated